keyboards/dasky/reverb/graphics/display.c

```diff
@@ -0,0 +1,137 @@
+// Copyright 2023 Dasky (@daskygit)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "display.h"
+#include "graphics/splash.qgf.h"
+#include "graphics/reverb.qgf.h"
+#include "graphics/robotomono20.qff.h"
+#include "qp_surface.h"
+#include <stdio.h>
+#include <math.h>
+
+static painter_image_handle_t reverb_logo;
+static deferred_token         display_task_token;
+static uint32_t               key_pressed_count = 0;
+
+static painter_device_t reverb_surface;
+static uint16_t         reverb_surface_fb[240 * 240];
```

Comment:
Not a big difference, but having a macro defined, i would use it
```suggestion
static uint8_t          reverb_surface_fb[SURFACE_REQUIRED_BUFFER_BYTE_SIZE(240, 240, 16)];
```

Reply:
resolved in d1f06cc1a9a63a0f4986c6f1fd5c078bd6651ad0